sktime/transformations/panel/tsfresh.py

```diff
@@ -560,7 +560,7 @@ def _get_selection_params(self):
 
         return selection_params
 
-    def fit_transform(self, X, y=None):
```

Comment:
If I am not mistaking, this is the part that is mostly equal to `fit`:

```python

self.reset()

        # skip everything if fit_is_empty is True and we do not need to remember data
        if self.get_tag("fit_is_empty") and not self.get_tag("remember_data", False):
            self._is_fitted = True
            return self

        # if requires_y is set, y is required in fit and update
        if self.get_tag("requires_y") and y is None:
            raise ValueError(f"{self.__class__.__name__} requires `y` in `fit`.")

        # check and convert X/y
        X_inner, y_inner = self._check_X_y(X=X, y=y)

        # memorize X as self._X, if remember_data tag is set to True
        if self.get_tag("remember_data", False):
            self._X = update_data(None, X_new=X_inner)
            
```

with some exceptions (see below)